packages/app/src/DBDashboardPage.tsx

```diff
@@ -247,7 +259,7 @@ const Tile = forwardRef(
                     color="gray"
                     size="xxs"
                     onClick={onEditClick}
-                    title="Alerts"
+                    title={alertTooltip}
```

Comment:
Optionally, it might be nice to convert this to a mantine `Tooltip` around the button instead of using the button title - I find the titles always require hovering too long to show up.